lib/usd/ui/layerEditor/mayaSessionState.cpp

```diff
@@ -223,10 +224,17 @@ void MayaSessionState::processNodeAdded(MObject& node)
 
 void MayaSessionState::proxyShapeAddedCBOnIdle(const MObject& obj)
 {
+    if (MFileIO::isNewingFile())
+        return;
+
     // doing it on idle give time to the Load Stage to set a file name
     // but we don't do a second idle because we could get a delete right after a Add
-    MDagPath dagPath;
-    MFnDagNode(obj).getPath(dagPath);
+    MDagPath   dagPath;
+    MFnDagNode dagNode;
+    if (!dagNode.setObject(obj))
+        return;
+
```

Comment:
While trying to fix the bug in various ways, I encountered a crash in Maya when calling getPath on a MDagNode that contained a null MObject.

In this case, it can happen when a new Maya scene is create, a stage added to it, then the scene delete before any on-idle task are processed. This can happen when writing scripts.